packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -79,21 +77,16 @@ abstract class CameraPlatform extends PlatformInterface {
     throw UnimplementedError('prepareForVideoRecording() is not implemented.');
   }
 
-  /// Starts a video recording.
-  ///
-  /// If no [file] parameter is provided, the recording will be saved to a new
-  /// file in a temporary location.
+  /// Starts a video recording and returns the file where it will be saved.
   ///
   /// The file is written on the flight as the video is being recorded.
-  /// If a file already exists at the path for the provided file instance,
-  /// an error will be thrown.
   /// The file can be read as soon as [stopVideoRecording] returns it.
```

Comment:
nit: the `stopVideoRecording` doesn't return anything, so we can remove `it` ;)

```suggestion
  /// The file can be read as soon as [stopVideoRecording] returns.
```